### PR TITLE
HW #13. Cypress actions

### PR DESCRIPTION
**Pull request by omgjulie**
Test for Login page was written using Cypress command action
Test for Smart Table was written using Page Object

### CODE REVIEW

File: cypress/e2e/tables/smartTable.cy.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    user.createNewUser(
+      userData.id,
+      userData.firstName,
+      userData.lastName,
+      userData.username,
+      userData.email,
+      userData.age,
+    );
+
+    cy.get("table tbody tr:first-child td:nth-child(2)").should(
```
Comment (sttaran) [inline cypress/e2e/tables/smartTable.cy.js]: you can also move checks to page object method

File: cypress/pageObjects/tables/SmartTable.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
+import { BasePage } from "../BasePage";
+
+export class SmartTable extends BasePage {
```
Comment (sttaran) [inline cypress/pageObjects/tables/SmartTable.js]: export class SmartTablePage extends BasePage

File: cypress/e2e/tables/smartTable.cy.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
+import { SmartTable } from "../../pageObjects/tables/SmartTable";
+
+describe("Table of users", () => {
+  it("should create and edit user", () => {
+    const user = new SmartTable();
```
Comment (sttaran) [inline cypress/e2e/tables/smartTable.cy.js]:
```
const smartTablePage = new SmartTablePage();
```

**Review by sttaran — CHANGES_REQUESTED** [review]

File: cypress/e2e/tables/smartTable.cy.js (hunk 1 above)
Comment (omgjulie) [reply cypress/e2e/tables/smartTable.cy.js]: created a new method **checkDataValidation**

**Pull request merged**